tools/generate_syscall_tables.sh

```diff
@@ -0,0 +1,162 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+# This script generates the syscall tables for seccompiler.
+
+set -e
+
+# Full path to the seccompiler tools dir.
+TOOLS_DIR=$(cd "$(dirname "$0")" && pwd)
+
+# Full path to the seccompiler sources dir.
+ROOT_DIR=$(cd "${TOOLS_DIR}/.." && pwd)
+
+# Path to the temporary linux kernel directory.
+KERNEL_DIR="${ROOT_DIR}/.kernel"
+
+generate_syscall_table_x86_64() {
+    path_to_rust_file="$ROOT_DIR/src/syscall_table/x86_64.rs"
+
+    echo "$1" > $path_to_rust_file
+
+    # the table for x86_64 is nicely formatted here: linux/arch/x86/entry/syscalls/syscall_64.tbl
+    cat linux/arch/x86/entry/syscalls/syscall_64.tbl | grep -v "^#" | grep -v -e '^$' |\
+        awk '{print $2,$3,$1}' | grep -v "^x32" |\
+        awk '{print "    map.insert(\""$2"\".to_string(), "$3");"}' | sort >> $path_to_rust_file
```

Comment:
Printing spaces ourselves to account for the Rust formatting is not future proof. Since we do not want the auto-generated file to be manually modified, we should consider running `cargo fmt` after the file is generated. This way we also don't need to manually print a number of spaces before writing the code.